crates/bootstrap/examples/cli.rs

```diff
@@ -45,7 +45,7 @@ struct Global {
     pub config: Config,
 }
 
-#[derive(serde_derive::Deserialize, Debug, smart_default::SmartDefault)]
```

Comment:
We should make a ticket to remove the `derive` feature from serde. https://github.com/serde-rs/serde/issues/2584